tests/integration/models/api-key.spec.ts

```diff
@@ -196,13 +245,17 @@ describe('API Key model', function () {
 		});
 
 		describe('balena.models.apiKey.revoke()', () => {
-			it('should be able to revoke an exising api key', function () {
-				return expect(balena.models.apiKey.revoke(this.apiKey.id))
-					.to.not.be.rejected.then(() => balena.models.apiKey.getAll())
-					.then(function (apiKeys) {
-						expect(apiKeys).to.be.an('array');
-						expect(apiKeys).to.have.lengthOf(0);
-					});
+			it('should be able to revoke an exising api key', async function () {
+				await expect(balena.models.apiKey.revoke(this.apiKey.id)).to.not.be
+					.rejected;
+				const apiKeys = await balena.models.apiKey.getAll({
```

Comment:
Move this to a common method? I see this repeated @ 48, 68 and here (251).

Reply:
Indeed... It's called `getAllNamedUserApiKeys()`...